quickstarts/dataingestion/dashboards/data_ingestion_breakdown.json

```diff
@@ -0,0 +1,986 @@
+{
+    "name": "Data Ingestion Breakdown",
+    "description": "",
+    "permissions": "PUBLIC_READ_WRITE",
```

Comment:
Permissions attribute needs to be removed @sri-shetty, same issue that I experienced.
cc/ @rudouglas @polfliet @aswanson-nr

Reply:
yes this should pass after removing this line
```suggestion
```

Reply:
Thank you @rudouglas and @Kav91 - Pushed PR again with the suggested changes.